Makefile

```diff
@@ -147,9 +147,14 @@ manifests: cmd/clusterctl/examples/aws/out/credentials ## Generate manifests for
 	kustomize build vendor/sigs.k8s.io/cluster-api/config/default/ >> cmd/clusterctl/examples/aws/out/provider-components.yaml
 
 .PHONY: manifests-dev
-manifests-dev: ## Push development manifest
+manifests-dev: dep-ensure dep-install binaries-dev ## Builds development manifests
 	MANAGER_IMAGE=$(DEV_MANAGER_IMAGE) $(MAKE) manifests
 
+.PHONY: binaries-dev
+binaries-dev: ## Builds and installs the binaries on the local GOPATH
+	go get -v ./...
+	go install -v ./...
```

Comment:
Why not use the existing bazel build targets here?

Reply:
That's a good question, I've been using the binaries on the GOPATH for `clusterctl` and `manager` so this would help in rebuilding those.

Reply:
Please add a comment that we should reconcile this with the bazel built binaries, otherwise lgtm

Reply:
Done :)